src/styles/spacing/input-width/index.njk

```diff
@@ -13,7 +13,8 @@ layout: layout-example.njk
   },
   classes: "govuk-!-width-full",
   id: "full-name",
-  name: "full-name"
+  name: "full-name",
+  autocomplete: "name"
```

Comment:
This is the only example that makes me a little nervous, just because it's such a generic use case (overriding input widths) that although I would expect most users to copy just the class name, I can see some users copying and pasting the example HTML and changing the question but leaving the `autocomplete` attribute in.

I wonder if we should change the example to something that wouldn't use an `autocomplete` attribute? What do you reckon?

Reply:
That's a good point. I'm struggling to think of a good alternative component instead of text input though - do you think just text would be a good enough example? 🤔

Reply:
I reckon so, we could probably do something similar to what we do for [fixed width inputs](https://design-system.service.gov.uk/components/text-input/#fixed-width-inputs) in the text input guidance?

Reply:
Sorry to be a pain, I'm starting to think we should just remove the autocomplete attribute in this example... Then it would match up with https://design-system.service.gov.uk/components/text-input/#fluid-width-inputs too. Seems like they should at least match?

Reply:
Oh, sorry, I think I misunderstood your last comment. I was suggesting we change the example to be more generic that wouldn't need an autocomplete, rather than suggesting using a different component. So I think we're on the same wavelength 🙂  Matching the fluid width examples makes sense to me – I hadn't scrolled down far enough to see them.

Reply:
@36degrees Sorry for the confusion! Added here: https://github.com/alphagov/govuk-design-system/pull/1766/commits/6bc3f954c202daa80e1b251fd59c29890e17c13b